src/qt/coincontroldialog.cpp

```diff
@@ -127,8 +127,8 @@ CoinControlDialog::CoinControlDialog(const PlatformStyle *_platformStyle, QWidge
     ui->treeWidget->headerItem()->setText(COLUMN_CHECKBOX, QString());
 
     ui->treeWidget->setColumnWidth(COLUMN_CHECKBOX, 84);
-    ui->treeWidget->setColumnWidth(COLUMN_AMOUNT, 110);
-    ui->treeWidget->setColumnWidth(COLUMN_LABEL, 190);
+    ui->treeWidget->setColumnWidth(COLUMN_AMOUNT, 130);
```

Comment:
To be consistent with https://github.com/bitcoin/bitcoin/blob/920c090f63f4990bf0f3b3d1a6d3d8a8bcd14ba0/src/qt/transactionview.h#L54-L61
```
ui->treeWidget->setColumnWidth(COLUMN_AMOUNT, 120);
ui->treeWidget->setColumnWidth(COLUMN_LABEL, 190); // do not touch
ui->treeWidget->setColumnWidth(COLUMN_DATE, 120);
```